docs/getting-started.md

```diff
@@ -107,14 +107,18 @@ Make sure to [update the React Native CLI to the latest version](https://github.
 
 #### iOS
 
-There is no additional configuration required on iOS except what follows in the next steps.
-
 If you're in a CocoaPods project (the default setup since React Native 0.60), make sure to install pods before you run your app:
 
 ```sh
 cd ios && pod install
 ```
 
+For React Native 0.60 or greater, add the library as the first import in your index.js file:
```

Comment:
The issue is happening on versions >= `0.61.0`.

Reply:
whoops. Fixed. Thank you.